README.md

```diff
@@ -271,6 +291,29 @@ End-to-end tests automatically runs on Travis CI when a PR is submitted. If you
 
 ## Known Issues and Workarounds
 
-_WIP_
+- If you are seeing the following error when installing with `helm install`, then make sure you have enabled at least one provider with `--set providers.vault.enabled=true` or `--set providers.azure.enabled=true`.
+
+  ```bash
+  Error: render error in "secrets-store-csi-driver/templates/required-check.yaml": template: secrets-store-csi-driver/templates/required-check.yaml:2:3: executing "secrets-store-csi-driver/templates/required-check.yaml" at <required "At least o...>: error calling required: At least one of the Values.providers is required to be enable
+  ```
+
+## Troubleshooting
+
+- To troubleshoot issues with the csi driver, you can look at logs from the `secrets-store` container of the csi driver pod running on the same node as your application pod:
+  ```bash
+  kubectl get pod -o wide
+  # find the secrets store csi driver pod running on the same node as your application pod
+
+  kubectl logs csi-secrets-store-secrets-store-csi-driver-7x44t secrets-store
+  ```
+- To troubleshoot issues with the provider component, you can look at logs from the `provider-log` container of the provider pod running on the same node as your application pod:
+  ```bash
+  kubectl get pod -o wide
+  # find the secrets store csi driver pod running on the same node as your application pod
```

Comment:
nit: find the secrets store csi driver provider pod running on the same node as your application pod